cms/djangoapps/contentstore/management/commands/tests/test_force_publish.py

```diff
@@ -0,0 +1,106 @@
+"""
+Tests for the force_publish management command
+"""
+import mock
+from django.core.management.base import CommandError
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from contentstore.management.commands.force_publish import Command
+from contentstore.management.commands.utils import get_course_versions
+
+
+class TestForcePublish(ModuleStoreTestCase):
```

Comment:
most of these tests don't actually change the course at all. You should use [`SharedModulstoreTestCase`](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/tests/django_utils.py#L219-L256). For the test that does change the default course, you can use the [`modified_courseware`](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/tests/django_utils.py#L297-L319) decorator.

This will just make tests run faster